nvme.c

```diff
@@ -5749,11 +5751,14 @@ static int format(int argc, char **argv, struct command *cmd, struct plugin *plu
 	}
 
 	if (!cfg.force) {
-		fprintf(stderr, "You are about to format %s, namespace %#x%s.\n",
+		fprintf(stderr,
+			"You are about to format %s, namespace %#x%s.\n",
 			dev->name, cfg.namespace_id,
-			cfg.namespace_id == NVME_NSID_ALL ? "(ALL namespaces)" : "");
+			cfg.namespace_id == NVME_NSID_ALL ? "(ALL namespaces)" :
+			"");
```

Comment:
I would leave the last line on the previous line, looks more readable to me.

We can tolerate a few more chars than 80 these days (IIRC, some parts of the kernel is 100 chars okay) especially if it makes it a bit more readable.